domain/learningandworkprogress/src/main/kotlin/uk/gov/justice/digital/hmpps/domain/learningandworkprogress/review/Exceptions.kt

```diff
@@ -7,7 +7,7 @@ class ReviewScheduleNotFoundException(val prisonNumber: String) :
   RuntimeException("Review Schedule not found for prisoner [$prisonNumber]")
 
 /**
- * Thrown when a Review Schedule already exists for a given Prisoner.
+ * Thrown when a prisoner already has an active Review Schedule. A Prisoner cannot have more than 1 active Review Schedules.
  */
-class ReviewScheduleAlreadyExistsException(val prisonNumber: String) :
-  RuntimeException("Review Schedule already exists for prisoner [$prisonNumber]")
+class ActiveReviewScheduleAlreadyExistsException(val prisonNumber: String) :
```

Comment:
Now that a prisoner can have many ReviewSchedules, we now have the concept of one (and only one) of them being the active one. You'll see this in the code below, but the "active" Review Schedule is considered the one that is not COMPLETED.
IE. a prisoner can have many COMPLETED Review Schedules, but only one that is not COMPLETED. This one record can be SCHEDULED or one of the EXEMPT_ statuses